intents/proto.go

```diff
@@ -1,5 +1,4 @@
 // Client
 //go:generate go run github.com/webrpc/webrpc/cmd/webrpc-gen -schema=intent.ridl -target=golang -pkg=intents -out=./intent.gen.go
-//go:generate go run github.com/webrpc/webrpc/cmd/webrpc-gen -schema=intent.ridl -target=typescript -out=./intent.gen.ts
```

Comment:
It look like this gets copied over to 
https://github.com/0xsequence/sequence.js/blob/55a0a1172cbfc0deb1f22e9cb9b327acd0e8f1ce/packages/waas/src/clients/intent.gen.ts#L6

...manually? cc @patrislav

Reply:
@david-littlefarmer so I propose to leave this line here

but remove the .gen.ts file from the repo via `.gitignore`